tests/facet_search.test.ts

```diff
@@ -98,6 +98,21 @@ describe.each([
     // @TODO: This is flaky, processingTimeMs is not guaranteed
     expect(response).toMatchSnapshot();
   });
+
+  test(`${permission} key: facet value search with exhaustive facet count`, async () => {
+    const client = await getClient(permission);
+
+    const params = {
+      facetName: "genres",
+      facetQuery: "a",
+      q: "Alice",
+      exhaustiveFacetCount: true,
+    };
+    const response = await client.index(index.uid).searchForFacetValues(params);
+
+    // @TODO: This is flaky, processingTimeMs is not guaranteed
```

Comment:
Same comment, I'm not sure I understand the issue here :)

Reply:
Snapshots rely on an idempotent output. Processing time is most of the times 1ms but sometimes 2 or more. Maybe this one single property can be ignored somehow, for instance by extracting it from the object before passing it to the snapshot assertion.